### PR TITLE
Fix some issus and add new config

### PR DESCRIPTION
**Pull request by Ph33rr**
Hello ,
add
theHarvester config
fix naabu config
fix xss
add new tag in nuclei
fix amass

### CODE REVIEW

**Review by Ph33rr — COMMENTED** [review]
Hello , 
add 
theHarvester config
fix naabu config
fix xss 
add new  tag in nuclei
fix amass

**Comment by yogeshojha** [comment]
Damn! This is greatly appreciated. I am reviewing this. Thank you :heart:

File: web/reNgine/tasks.py [hunk 1]
```diff
@@ -988,9 +990,9 @@ def port_scanning(
         naabu_command = naabu_command + \
             ' -rate {}'.format(
                 yaml_configuration[PORT_SCAN][NAABU_RATE])
-
+            #new format for naabu config
     if USE_NAABU_CONFIG in yaml_configuration[PORT_SCAN] and yaml_configuration[PORT_SCAN][USE_NAABU_CONFIG]:
-        naabu_command += ' -config /root/.config/naabu/naabu.conf'
+        naabu_command += ' -config /root/.config/naabu/config.yaml'
```
Comment (yogeshojha) [inline web/reNgine/tasks.py:995]: Oopsie! Thank you for pointing this out!

**Comment by yogeshojha** [comment]
Thank you for your contributions. This means a lot for the community.

**Pull request merged**